### PR TITLE
Create Community Pharmacy Screening Form

### PR DESCRIPTION
**Pull request by ckote**
Screening form for cp model

### CODE REVIEW

**Review by RAJABIBRAZ — APPROVED** [review]

**Review by makombe — COMMENTED** [review]
@ckote Do we have the metadata for this form existing? or you have just generated the form uuid? Thanks

File: configuration/ampathforms/Community Pharmacy Screening Form [hunk 1]
```diff
@@ -0,0 +1,1300 @@
+{
+  "name": "Community Pharmacy Screening Form",
+  "description": "ROC Screening Tool Community Pharmacy Form",
+  "version": "2.0",
+  "published": true,
+  "uuid": "f7dbe6b6-2a5a-46e6-af52-45bf2962f4aa",
+  "retired": false,
+  "encounter": "Consultation",
```
Comment (makombe) [inline configuration/ampathforms/Community Pharmacy Screening Form:8]: We should have a proper encounter type for this form other than using consultation

**Comment by ckote** [comment]
@makombe @patryllus confirmed that he had created the metadata. The form is as shared by Wang'oo

File: configuration/ampathforms/Community Pharmacy Screening Form (hunk 1 above)
Comment (makombe) [inline configuration/ampathforms/Community Pharmacy Screening Form:8]:
```suggestion
  "encounter": "CPM Screening Encounter",
```

Comment (makombe) [reply configuration/ampathforms/Community Pharmacy Screening Form:8]: @ckote correct the encounter type has indicated above. Thanks

**Review by makombe — APPROVED** [review]

**Pull request merged**